_posts/2023-01-03-do-you-really-need-more.md

```diff
@@ -0,0 +1,128 @@
+
+Customer habits have also changed: user behavior, smartphone usage, user experience standards, etc. People are more likely to place orders from their phones, and merchants use social media to become visible to their ideal customers and bring more qualified traffic to their storefront.
+
+Today's user experience must meet the standards set by the most popular online services. Design, interactions, and performance expectations from users are completely different than they were 10 years ago. This change also affects SEO; search engines prioritize websites that meet performance and user experience standards, especially on mobile.
+
+**For this reason, I believe merchants who are still selling online today are those who embraced uncertainty and continually challenged themselves.**
+
+### Technology has continued to evolve
+
+On the web, technology moves fast and new improvements or new standards appear continuously.
+
+As an example, Magento store owners have to deal with:
+
+- new PHP versions (PHP is in version 8.2 as I write these lines)
+- libraries getting deprecated (did you know that several libraries used by Magento are unmaintained by their initial owners, and are now part of the Magento core?)
+- open-source projects die (do you remember OSCommerce? Also, the “Magento 1” project is now dead… but open-source allows projects such as Magento-LTS to write a new story on top of this legacy)
+
+Some of these technology changes are welcome (i.e: new PHP versions bring better performance and code maintainability for PHP projects) but others are seen as a burden (i.e: a JS dependency is so entangled in many different parts of the frontend that replacing it with another is a huge task to undertake).
+
+**This is where I see the interest of a solution such as MagentoLTS.**
+It is a lighthouse that remains stable (in terms of features) while following the needed technology changes from the underlying ecosystem.
+Thanks to the OpenMage community, merchants can maintain their fully tailored online shop up-to-date without having to recreate their custom features on a new foundation.
+
+This is perfect for features that are *just done*!
+But for features that must evolve with users standards and usage, I think it often is not satisfying enough.
+
+### The rise of 3rd-party services
+
+External services rise and disappear. Embracing this fact is a key factor allowing merchants to innovate and grow faster than their competitors.
+
+Do you want to grow internationally? Support new payment and shipping methods used by customers from different countries.
+Do you want to sell on a marketplace? Export your catalog to the marketplace your targeted customers visit daily.
+Do you want to engage your community? Embed widgets from your main social media platforms on your website, and optimize your pages for sharing and reposting.
```

Comment:
Good catch!\
I wanted some line breaks, but Jekyll's markdown parser uses double spaces `  ` instead of `\` … and my editor autoformat removed them without me noticing.

I'll replace with `<br>` tags:
```suggestion
Do you want to grow internationally? Support new payment and shipping methods used by customers from different countries.<br>
Do you want to sell on a marketplace? Export your catalog to the marketplace your targeted customers visit daily.<br>
Do you want to engage your community? Embed widgets from your main social media platforms on your website, and optimize your pages for sharing and reposting.
```